### PR TITLE
Test downlads functions (remote data)

### LINKED ISSUE #41: Test remote data access
Hello,

We should add unit tests to test remote data access.

If a remote data access is broken we should depreciate it and document this.

Kind regards

### PR DESCRIPTION
**Pull request by femtotrader**
Closes #41

### CODE REVIEW

**Comment by femtotrader** [comment]
@iblis17 @milktrader or @GordStephen Could you please review?

File: src/downloads.jl [hunk 1]
```diff
@@ -28,6 +28,7 @@ See Also
 """
 
 function yahoo(data::String="^GSPC")
+    Base.depwarn("Yahoo Finance API changed, this function may not work anymore", :yahoo)
```
Comment (iblis17) [inline src/downloads.jl]: IIRC, yahoo API is shut down.

File: test/downloads.jl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    @testset "FRED" begin
+        ta = fred()
+        @test length(ta.timestamp) > 100
+    end
+
+    @testset "Yahoo" begin
+        # ta = yahoo()
+        # @test length(ta.timestamp) > 100
+        @test_broken 1==2
+    end
```
Comment (iblis17) [inline test/downloads.jl]: so, remove this code block?

File: src/downloads.jl (hunk 1 above)
Comment (femtotrader) [reply src/downloads.jl]: Not so clear...
See `https://github.com/pydata/pandas-datareader/issues/549`

File: test/downloads.jl (hunk 2 above)
Comment (femtotrader) [reply test/downloads.jl]: No I prefer to keep it and mark this test as broken until `yahoo` function exists.
We could remove test block when/if `yahoo` function doesn't exist anymore.

Comment (iblis17) [reply test/downloads.jl]: okay.

**Pull request merged**